types/react-responsive-masonry/index.d.ts

```diff
@@ -0,0 +1,41 @@
+// Type definitions for react-responsive-masonry 2.1
+// Project: https://github.com/cedricdelpoux/react-responsive-masonry#readme
+// Definitions by: themashcodee <https://github.com/themashcodee>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { CSSProperties, ReactNode, ReactElement } from 'react';
+
+export interface MasonryProps {
+    children: ReactNode;
+    /**
+     * Injected by ResponsiveMasonry
+     *
+     * Default Value = 3
+     */
+    columnsCount?: number;
+    /**
+     * Margin surrounding each item e.g. "10px" or "1.5rem" (css gap property)
+     *
+     * Default Value = "0"
+     */
+    gutter?: string;
+    className?: string;
+    style?: CSSProperties;
+}
+
+export interface ResponsiveMasonryProps {
+    children: ReactNode;
```

Comment:
same here:
https://github.com/cedricdelpoux/react-responsive-masonry/blob/master/src/ResponsiveMasonry/index.js#L72
```suggestion
    children: ReactNode | ReactNode[];
```

Reply:
skip this, I'm stupid :)